src/utility.jl

```diff
@@ -240,26 +238,27 @@ imshow(result)
 evaluation methodology for optical flow.International Journal of Computer Vision, 92(1):1–31, 2011.
 """
 function evaluate_flow_error(ground_truth_flow::Array{SVector{2, Float64}, 2}, estimated_flow::Array{SVector{2, Float64}, 2}, error_type::AngularError)
-    result = Array{Float64, 2}(undef, size(estimated_flow))
-    for i in 1:size(estimated_flow)[1]
-        for j in 1:size(estimated_flow)[2]
-            if is_flow_known(estimated_flow[i, j]) && is_flow_known(ground_truth_flow[i, j])
-                p1 = append!(convert(Array{Float64, 1}, ground_truth_flow[i, j]), 1.0)
-                p2 = append!(convert(Array{Float64, 1}, estimated_flow[i, j]), 1.0)
-                cosine = dot(p1, p2)/(norm(p1)*norm(p2))
-                if cosine > 1
-                    result[i, j] = 0
-                else
-                    result[i, j] = acos(cosine)
-                end
-                if result[i, j] <= 1e-5
-                    result[i, j] = 0
-                end
+    result = similar(Array{Float64, 2}, axes(estimated_flow))
+    for i in CartesianIndices(estimated_flow)
+        p1 = ground_truth_flow[i]
+        p2 = estimated_flow[i]
+        if is_flow_known(estimated_flow[i]) && is_flow_known(ground_truth_flow[i])
```

Comment:
I think you accidentally missed this one...
```julia
if is_flow_known(p1) && is_flow_known(p2)
```

Reply:
Sorry!